### PR TITLE
Add gitattributes file

### PR DESCRIPTION
**Pull request by dereuromark**
Resolves https://github.com/rectorphp/rector/issues/791

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 /vendor
 /build
 composer.lock
+.idea/
```
Comment (TomasVotruba) [inline .gitignore]: Why is this one needed?

Comment (dereuromark) [reply .gitignore]: As most people use phpstorm, this is just annoying to have not in this file.
Doesnt hurt anyone else so a recommended addition in general.

Comment (TomasVotruba) [reply .gitignore]: Ok, fine with me :+1:

**Comment by TomasVotruba** [comment]
Thank you @dereuromark

**Pull request merged**